ports/pcre/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "pcre",
+  "version-string": "8.45",
```

```suggestion
  "version": "8.45",
```
It might be better to use `version `here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version.

---

OK, updated to use `version` instead of `version-string`.